charmhelpers/fetch/ubuntu.py

```diff
@@ -181,6 +181,11 @@
 CMD_RETRY_DELAY = 10  # Wait 10 seconds between command retries.
 CMD_RETRY_COUNT = 3  # Retry a failing fatal command X times.
 
+CONTAINS_CIDR = re.compile(r'.+?[0-9:]+\/[0-9]')
+CIDR_WARNING = ('Passing NO_PROXY string that includes a cidr. '
```

Comment:
Maybe we should also check for the wildcard syntax with hostnames, see https://bugs.launchpad.net/juju/+bug/1773463